src/components/SearchResultDetail/SearchResultDetail.tsx

```diff
@@ -32,6 +58,26 @@ const SearchResultDetail = ({ id }: { id: string }) => {
     findSearchResult();
   }, [id]);
 
+  const downloadPdf = async () => {
+    try {
+      setDownloadInProgress(true);
+      const pdfString = await exportSearchResultPdfAction(searchResult.id);
+      if (!pdfString) {
+        throw new Error('The PDF file cannot be exported');
+      }
+
+      const fileName = `${searchResult.keyword}-${searchResult.searchEngine}-result.pdf`;
+      generateDownloadLink({ base64String: pdfString, fileName, mimeType: MIME_TYPE_PDF });
+      setDownloadInProgress(false);
+
+      toast(`PDF export successfully created: ${fileName}.`, 'success');
```

Comment:
```suggestion
      toast(`PDF export successfully created: ${fileName}`, 'success');
```

As the error toast message not have the dot 🙈